lib/grape-swagger/endpoint.rb

```diff
@@ -227,11 +227,11 @@ def apply_success_codes(route)
       [default_code]
     end
 
-    def tag_object(route)
+    def tag_object(route, path)
       version = GrapeSwagger::DocMethods::Version.get(route)
       version = [version] unless version.is_a?(Array)
       Array(
-        route.path.split('{')[0].split('/').reject(&:empty?).delete_if do |i|
+        path.split('{')[0].split('/').reject(&:empty?).delete_if do |i|
```

Comment:
I know you don't changed that, but please can we extract this split and reject into a separate method which explains, what it did? It's really hard to understand and a good method name would help with that.